README.md

```diff
@@ -76,7 +76,8 @@ This is the file under `docs/api/accounts/profile.json`:
     "examples": {
         "OK": {
             "request": {
-                "url": "/accounts/me"
+                "url": "/accounts/me",
+                "method": "GET"
```

Comment:
To be honest, these two are *inherited* from the top level endpoint definition and don't need to be duplicated here, since the URL is already a valid one and not a pattern (e.g. `'/accounts/{someone}/'`).